src/v1/transaction.rs

```diff
@@ -14,6 +15,12 @@ use num_bigint::BigInt;
 #[cfg(feature = "serde")]
 use serde::{Deserialize, Serialize};
 
+use super::address::StakingCredential;
+use super::crypto::PaymentPubKeyHash;
+use super::datum::Datum;
+use super::tuple::Tuple;
+use super::value::CurrencySymbol;
+
```

Comment:
[nit] It kind of feels odd that some modules are imported as `crate::v1::xyz` while some are `super::xyz`, could you change the all of the former to use super as well? Also, could you remove the empty line, to let the formatter auto-sort all imports?

Reply:
My editor did auto-import for me so I really don't know. Also, it seems that `cargo fmt` cannot combine imports, which is a bit inconvenient.

Reply:
Anyway, I have replaced all the `crate::v1/v2::something` with `super::something`, but import sorting doesn't seem to work whenever there are `#[cfg(feature = "something")]` lines among the `use` lines.